mithril-aggregator/src/database/record/signed_entity.rs

```diff
@@ -233,6 +256,50 @@ impl TryFrom<SignedEntityRecord> for SnapshotListItemMessage {
     }
 }
 
+impl TryFrom<SignedEntityRecord> for CardanoStakeDistributionMessage {
+    type Error = StdError;
+
+    fn try_from(value: SignedEntityRecord) -> Result<Self, Self::Error> {
+        #[derive(Deserialize)]
+        struct TmpCardanoStakeDistribution {
```

Comment:
Since it's an internal structure within the function and it's named this way in all other conversion functions, we'll leave it as is. We can rename it if necessary in another PR.